smapply/serializers.py

```diff
@@ -0,0 +1,10 @@
+"""Serializers for smapply"""
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.Serializer):
+    """Serializer for SMApply user data"""
+    email = serializers.CharField()
+    first_name = serializers.CharField(allow_blank=True)
```

Comment:
I think the first name and last name should be required.

Reply:
Should be required now.